.github/workflows/publish.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [arm64-secure, ubuntu-20.04]
+        os: [arm64-secure, ubuntu-20.04, ubuntu-latest]
```

Comment:
Is there a reason for this? Does cross-building not work on ubuntu-20.04 (I don't even know what version of ubuntu "latest" will give; I imagine it isn't consistent?)

Reply:
I believe ubuntu-latest is a moving target. I am with Avi here, we should not depend on `latest` for anything at all. It works now, but in a month o so it might suddenly throw a random error on random machines of random developers (depending on the cache state). Fun to address

Reply:
Sure, but right now we don't have a choice -- what we need is available only there and we have 0 control over how GitHub manages its infrastructure. The argument "ubuntu-latest is a moving target" is sadly applicable to `ubuntu-20.04` as well -- since GH updates it quite frequently.

IOW, if we want something for free (GH infra) we need to roll with the punches -- if we want full control we need to pay for it (which we're currently in no position to afford).

Reply:
True, `20.04` is moving, but it is moving in a much narrower band than `latest`. What is the current `latest`? Can we just use that?